contracts/contracts/ERC677/linkToken.cairo

```diff
@@ -0,0 +1,38 @@
+%lang starknet
```

Comment:
We need to define a convention how do we actually name our contracts, as we currently use all of them: camelCase, PascalCase, snake_case. Which one is it? :) cc @archseer

Reply:
done

Reply:
What is exactly done here? We currently have functions like `IERC677.transferAndCall` and `IERC677Receiver.on_token_transfer`.

Consider this discussion: https://github.com/OpenZeppelin/cairo-contracts/discussions/34

Reply:
I've opened an GH Issue on this: https://github.com/smartcontractkit/chainlink-starknet/issues/80

cc @archseer